README.md

```diff
@@ -15,7 +15,7 @@ back and forth between your real source code and the browser view.
 * **Do it yourself.** Because Brackets is open source, and built with HTML, CSS
 and JavaScript, you can [help build](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md) the best code editor for the web.
 
-Brackets is at 1.0 and we're not stopping there. We have many feature ideas on our
+Brackets is at 1.2 and we're not stopping there. We have many feature ideas on our
```

Comment:
I don't think we want to update this for every release. I think the intent is more like "Brackets is at version 1 and we're not stopping there."